.gitlab-ci-main.yml

```diff
@@ -96,25 +88,28 @@ variables:
 #
 
 vet:
+  <<: *static_template
+  stage: static
   script:
     - composer validate
   when: on_success
   allow_failure: false
-  stage: static
 
 lint:
   <<: *static_template
   extends: .job-lint
   # Run all lines together so that they all run even if any fail.
-  script: |
-    set +exuo pipefail # Linting...
-    docker-compose exec -T test bash -c '/app/vendor/bin/govcms-lint web/modules/custom'
-    docker-compose exec -T test bash -c '/app/vendor/bin/govcms-lint web/themes/custom'
+  script:
+    - |
+      set +exuo pipefail # Linting...
+      ./vendor/bin/govcms-lint web/modules/custom
+      ./vendor/bin/govcms-lint web/themes/custom
+
 unit:
   <<: *static_template
   extends: .job-unit
   script:
-    - docker-compose exec -T test bash -c './vendor/bin/govcms-unit --testsuite=unit'
+    - ./vendor/bin/govcms-unit --testsuite=unit'
```

Comment:
Stray `'` there at the end 🙂 ...which is ironic, cause linting might have picked it up 😅

Reply:
Thanks greg!